package.json

```diff
@@ -9,8 +9,9 @@
     "start": "next start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "lint": "eslint . --ext=tsx,ts,js,jsx",
-    "lint:fix": "yarn lint --fix",
+    "lint": "yarn lint:eslint && prettier --check '**/*.(yml|json)'",
+    "lint:eslint": "eslint . --ext=tsx,ts,js,jsx",
+    "lint:fix": "yarn lint:eslint --fix && prettier --write '**/*.(yml|json)'",
```

Comment:
This is unfortunately the kind of thing the current setup wanted to avoid. It's odd that eslint would be doing the same work for the file extensions provided as prettier is doing in the one-off cli calls.

Reply:
Salam @nabeel- et al, I referenced plugin owner's comments in #33, but to repeat:

> You shouldn't pass non-js files into eslint unless you've got a plugin to explicitly handle that (and even then interaction with this plugin gets a little weird). Instead you should run prettier over your json files in a separate step instead of trying to smush it into your eslint workflow... eslint-plugin-prettier aims to be just a rule - it deliberately doesn't want to start wrangling ESLint preprocessors as that's out of scope for what I want it to to care about

https://github.com/prettier/eslint-plugin-prettier/issues/294

In other words: *ESlint is for code, not config and other things, while prettier is more general.*  So if you want something cleaner, your going to have to write your own pagackge/scripts to wrap around eslint and prettier together to achieve a 'nice' interface.

In conclusion: What I've proposed is not overly long/ugly in terms of scripts.  But if other folks disagree now or after adding more stuff and would like to seem something cleaner then a next simple step would be some scripts with nice code comments (e.g. ./scripts/format-files) to clean up package.json